src/content/release-notes.mdx

```diff
@@ -5,6 +5,42 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.27.0
+
+9 December 2024
+
+This version is compatible with Kubernetes versions 1.27 to 1.30
+
+### Breaking Changes
+
+- Remove code for our deprecated Quickstarts feature <!-- 8842 -->
+
+### New Features
+
+- We now allow the ability for admins of Okteto to [set the minimum supported CLI version for their team](admin/dashboard.mdx#command-line-cli)  <!-- 8829 -->
+- We've published the Okteto Manifest JSON Schema for inline suggestions and validation for creating and editing Okteto manifests within your code editor
+- Added support for overwriting CLI images at the Helm chart level
+- Shipped additional support for the installation of [Okteto in Air-Gapped Environments](self-hosted/manage/air-gapped.mdx) <!-- 8914 -->
+- Configured buildkit probes to only accept requests when buildkit is healthy  <!-- 8871 -->
+- Added support for configuring the Okteto control plane jobs TTL at the helm level  <!-- 8917 -->
+- feat: send cli image on graphql clustermetadata  <!-- 8914 -->
+
+### Improvements
+
+- We've reorganized the [Admin Dashboard menu items](admin/dashboard.mdx) into groups for easier navigation  <!-- 8840 -->
+- Added readonly tags in the Dashboard to Catalog, Cloud Credentials, and Registry Credential items that were created via CRDs  <!-- 8886 -->
+- Updated buildkit cacheRatio value to `0.8`  <!-- 8921 -->
+- feat: `wake`, `sleep` & `destroyall` rootless (PLAT-738)  <!-- 8919 -->
```

Comment:
In the next two items I would say that we have migrated all the images of the control plane to be rootless to improve the overall security of Okteto

Reply:
Agree, I would use a single point for this. Because it affects to all the components

Reply:
@pchico83 Should we mention also that installer image is no longer needed? In case someone might be specifying that image for whatever reason

Reply:
yes, makes sense. we could also mention the cli images that have been combined in `okteto/okteto`